LS-184-helm-doc (#30)
datastax/pulsar-docs: modules/ROOT/pages/helm-chart-config.adoc

= Helm Chart Configuration Values

> This page will explain configuration values included in the Helm chart https://github.com/datastax/pulsar-helm-chart/blob/master/helm-chart-sources/pulsar/values.yaml[values file].
> 
> You can override chart configurations during installation as necessary if you’re an advanced user, or make changes after a default installation using `helm upgrade` at a later time.

Reviewer: ```suggestion
This page explains configuration values included in the Helm chart https://github.com/datastax/pulsar-helm-chart/blob/master/helm-chart-sources/pulsar/values.yaml[values file].

If you are an advanced user, you can override chart configurations during installation as necessary or make changes after a default installation using `helm upgrade` at a later time.
```

For example, when using `helm install` to install Luna Streaming, you can point to the example configuration file using the `-f` flag:

----
helm install pulsar -f dev-values.yaml datastax-pulsar/pulsar
----

## Values

[cols=4*]
|===
|*Key*
|*Type*
|*Default*
|*Description*

// | Key | Type | Default | Description |
// |-----|------|-------------|------------------|
// #| cassandra.enabled | bool | `true` | Enables installation of Cassandra cluster. Set to false if you only wish to install operators. |
| fullnameOverride | string | `pulsar` | Overrides the name of the helm deployment 
| dnsName | string | `pulsar.example.com` | DNS name for load balancer 

> | rbac | bool | `true` | use `ClusterRole` and `ClusterRoleBinding` resources when set to `true``, use namespace Role and RoleBinding resources when set to `false` 

Reviewer: ```suggestion
| rbac | bool | `true` | Uses `ClusterRole` and `ClusterRoleBinding` resources when set to `true`; uses `NamespaceRole` and `RoleBinding` resources when set to `false` 
```
